src/coredata.cs

```diff
@@ -640,22 +767,48 @@ public interface NSManagedObject {
 		void DidTurnIntoFault ();
 
 		[Export ("valueForKey:")]
+		[return: NullAllowed]
+#if XAMCORE_4_0
+		NSObject ValueForKey (string key);
```

Comment:
rename to `GetValue`, the current API is missing verb/action and it's stuff is not helpful.